ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>This next step can take a while. The client library prints out statements that include the name of the new dataset--you might want to copy the resource name down someplace.</blockquote><p>Suggest maybe: "Creating and importing data is a long-running operation. This next step can take a while. The sample waits for the operation to complete, outputting statements as the operation progresses. The statements contain the full name of the dataset that you will use in the following section.</p><p> </p><blockquote><strong>Note</strong>: You can close this tab while you wait for this operation to complete.</blockquote><p>Can you clarify what you mean by this "tab"? In my testing, I don't see separate tabs here. (same comment for the other operations)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<blockquote><strong>Note</strong>: You can close noteboook while </blockquote><p>You can close the notebook while (added "the")</p>

Reply:
<p>Done.</p>